L1Trigger/RPCTriggerPrimitives/plugins/BuildFile.xml

```diff
@@ -0,0 +1,36 @@
+<library file="*.cc" name="L1TriggerRPCTriggerPrimitivesPlugins">
+  <use name="L1Trigger/RPCTriggerPrimitives"/>
+  <flags EDM_PLUGIN="1"/>
+</library>
+
+<use name="FWCore/Concurrency"/>
+<use name="FWCore/MessageLogger"/>
+<use name="FWCore/Framework"/>
+<use name="FWCore/ParameterSet"/>
+<use name="FWCore/PluginManager"/>
+<use name="FWCore/Utilities"/>
+
+<use name="DataFormats/Common"/>
+<use name="DataFormats/DetId"/>
+<use name="DataFormats/MuonDetId"/>
+<use name="DataFormats/L1TMuon"/>
+<use name="DataFormats/RPCDigi"/>
+<use name="DataFormats/RPCRecHit"/>
+<use name="DataFormats/DTRecHit"/>
+<use name="DataFormats/CSCRecHit"/>
+<use name="DataFormats/TrackingRecHit"/>
+
+<use name="CondFormats/RPCObjects"/>
+<use name="CondFormats/DataRecord"/>
+
+<use name="L1Trigger/L1TMuon"/>
+<use name="L1Trigger/L1TMuonEndCap"/>
+
```

Comment:
where are these tow L1Trigger packages used? Removing them seems to have no effect.

Reply:
Deleted